cpr/cookies.cpp

```diff
@@ -1,51 +1,97 @@
 #include "cpr/cookies.h"
 
 namespace cpr {
-std::string Cookies::GetEncoded(const CurlHolder& holder) const {
+const std::string Cookie::GetDomain() const {
+    return domain_;
+}
+
+bool Cookie::IsIncludingSubdomains() const {
+    return includeSubdomains_;
+}
+
+const std::string Cookie::GetPath() const {
+    return path_;
+}
+
+bool Cookie::IsHttpsOnly() const {
+    return httpsOnly_;
+}
+
+time_t Cookie::GetExpires() const {
```

Comment:
`time_t` is C. Please change this to something like a [`std::chrono::time_point`](https://en.cppreference.com/w/cpp/chrono/time_point). While doing this, please ensure it's compatible with our current cpp standard.

Reply:
Done!